azure-pipelines.yml

```diff
@@ -8,16 +8,16 @@ variables:
     - group: SDL_Settings
   - ${{ if eq(variables['System.TeamProject'], 'public') }}:
     - name: PoolProvider
-      value: NetCore1ESPool-Public
+      value: NetCorePublic-Pool
```

Comment:
I don't think we need to revert the non-scheduled runs, those are the only ones I've seen failing, because those are the only ones that run in int. I'm fine with the full revert if that's easier for you though.

Reply:
It would be more tricky to revert it partially because the image names differ between Helix and 1ES pools.